examples/java/src/main/java/org/apache/beam/examples/spanner/SpannerCSVLoader.java

```diff
@@ -0,0 +1,143 @@
+/*
```

Comment:
I would not put this in the global examples – we're trying to keep this independent of specific proprietary clouds. Instead, we'd probably like to use it as part of an integration test in the `sdks/java/io/google-cloud-platform-java` module.